src/diffusers/models/resnet.py

```diff
@@ -256,7 +256,7 @@ def __init__(
                 "This class cannot be used with `time_embedding_norm==spatial`, please use `ResnetBlockCondNorm2D` instead",
             )
 
-        self.pre_norm = True
+        self.pre_norm = pre_norm
```

Comment:
Is `prep_norm` defaulted to True?

Reply:
Yes, the default value of `prep_norm` in ResnetBlock2D is True.

Also, for unet_2d_blocks and unet_3d_blocks, which use ResnetBlock2D, the default value of `resnet_pre_norm` are True.
https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/unets/unet_2d_blocks.py
https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/unets/unet_3d_blocks.py